src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```
@@ -53,6 +53,8 @@ The .NET Foundation licenses this file to you under the MIT license.
       <EventPipeName Condition="'$(EventSourceSupport)' == 'true'">libeventpipe-enabled</EventPipeName>
 
       <LinkStandardCPlusPlusLibrary Condition="'$(LinkStandardCPlusPlusLibrary)' == '' and '$(_IsiOSLikePlatform)' == 'true' and '$(InvariantGlobalization)' != 'true'">true</LinkStandardCPlusPlusLibrary>
+      <ClrgcSupportName>libclrgc-disabled</ClrgcSupportName>
+      <ClrgcSupportName Condition="'$(IlcStandaloneGCSupport)' == 'true'">libclrgc-enabled</ClrgcSupportName>
```

```suggestion
      <StandaloneGCSupportName>libstandalonegc-disabled</StandaloneGCSupportName>
      <StandaloneGCSupportName Condition="'$(IlcStandaloneGCSupport)' == 'true'">libstandalonegc-enabled</StandaloneGCSupportName>
```
To match `IlcStandaloneGCSupport` name?